_docs/Qabel-Protocol-Register.md

```diff
@@ -0,0 +1,42 @@
+---
+title: "Protocol: register"
+---
+# Register Specification
+
+## Abstract
+
+Protocols to store and search information about the users' identities on a server.
```

Comment:
In Components-Register.md its "user's" (singular) here it is "users'" (plural) wouldn't it be better to keep it consistend?